test/config/nnictl_experiment/test_import.json

```diff
@@ -0,0 +1,4 @@
+[
+    {"parameter": {"C": 0.15940134774738896, "kernel": "sigmoid", "degree": 3, "gamma": 0.07295826917955316, "coef0 ": 0.0978204758732429}, "value": 0.6},
```

Comment:
`"coef0 "`?

Reply:
Because in `search_space.json` is `"coef0 "`, should I modify them all?
https://github.com/microsoft/nni/blob/1d8b8e48e2f3b5ffe8336cfb5d5e3e6b06b24ad2/examples/trials/sklearn/classification/search_space.json#L6